src/main/java/mate/academy/model/Order.java

```diff
@@ -0,0 +1,69 @@
+package mate.academy.model;
+
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import jakarta.persistence.FetchType;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.OneToMany;
+import jakarta.persistence.OneToOne;
+import jakarta.persistence.Table;
+import java.time.LocalDateTime;
+import java.util.List;
+
+@Entity
+@Table(name = "orders")
+public class Order {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private long id;
+    @OneToOne
+    private User user;
+    @Column(name = "order_date")
+    private LocalDateTime orderDate;
+    @OneToMany
```

Comment:
The `@OneToMany` relationship with `tickets` should specify `FetchType.LAZY` to avoid unnecessary data loading, which can lead to performance issues. Additionally, ensure that the `mappedBy` attribute is set to indicate the owner of the relationship.